frontend/javascripts/libs/toast.tsx

```diff
@@ -141,5 +168,39 @@ const Toast = {
   close(key: string) {
     notification.destroy(key);
   },
+  message(
+    type: ToastStyle,
+    message: ToastMessageCallback | React.ReactNode = "Success :-)",
+    config: ToastConfig = {},
+    details?: string | undefined,
+  ) {
+    renderIndependently((destroy) => {
+      // Deferring destroy to give the Notification API a chance to close the toast via timeout
+      // before unmounting the parent component created with renderIndependently.
```

Comment:
please mention that this avoids an error in the code comment. I don't understand 100% why, though. Won't there still be two parties who try to destroy the toast? and the latter one will "lose" as it's already destroyed?

Reply:
ah ok, now I understand :) antd tries to close the inner toast and `destroy` destroys the outer parent component. therefore, calling `destroy` within onClose is too early, because the toast is not torn down yet. somehow I assumed that both parties would try to destroy the same thing, but they refer to parent VS child component.
knowing that, the comment looks good :sweat_smile: